src/Polymesh.ts

```diff
@@ -214,16 +214,12 @@ export class Polymesh {
     return transferPolyX.prepare(args, this.context);
   }
 
-  // TODO: uncomment for v2
-  /*
-   * Get the POLYX balance of the current account
-   * NOTE: We don't expose this method for Testnet v1
+  /**
+   * Get the POLYX balance of an identity
    */
-  /*
-  public getIdentityBalance(): Promise<BigNumber> {
-    return this.context.getCurrentIdentity().getPolyXBalance();
+  public getIdentityBalance(args?: { did: string }): Promise<BigNumber> {
```

Comment:
I know that if you have the Identity, you could call directly to getPolyXBalance or getTokenBalance. But should `did` support `Identity` entity too? To keep consistence.

Reply:
Agreed. Will fix